<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrelievoController</class>
 <widget class="QDialog" name="PrelievoController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_prelievo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Digitare l'importo della Spesa:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Immettere il PIN:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>mostra PIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errori">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>325</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>325</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Data della Spesaa:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria della Spesa:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
